src/com/owncloud/android/lib/common/Quota.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Nextcloud Android client application
+ *
+ * @author Mario Danic
+ * Copyright (C) 2017 Nextcloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.owncloud.android.lib.common;
+
+import org.parceler.Parcel;
+
+/**
+ * Created by mdjanic on 24/01/2017.
+ */
+
+@Parcel
+public class Quota {
+    public long free;
+    public long used;
+    public long total;
+    public long quota;
+    public double relative;
```

Comment:
Yes. If they're private, then we incur a significant performance penalty due to reflection.